src/qcodes/instrument_drivers/Keysight/P5004B.py

```diff
@@ -0,0 +1,7 @@
+from .Keysight_P5004B import KeysightP5004B
+
+
+class P5004B(KeysightP5004B):
+    """
+    Alias for backwards compatibility.
+    """
```

Comment:
Since this driver has never existed in qcodes there is no need for creating this since there is no backwards compatibility. Please remove this file